config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/NanoForgeTier4-hUEbNbngSA6JxT_Pjq8DNw==.json

```diff
@@ -8,15 +8,15 @@
   "properties:10": {
     "betterquesting:10": {
       "autoClaim:1": 0,
-      "desc:8": "The Transcendent Nanite has the required speed and complexity to create even the most complex nanites... but that doesn\u0027t necessarily insinuate that it is sufficient for your demands. You need a nanite that will perform at peak efficacy; a nanite that will outperform all other nanites to aid your own nanite production. Clearly then you must choose the Eternity Nanite! But you suspect that you\u0027ll need a far more complicated structure to be able to contain such a process.\n\n[note]Does not allow for any new recipes, but allows a greater speed at the cost of assembling a more expensive structure.[/note]",
+      "desc:8": "The Transcendent Nanite has the required speed and complexity to create most complex nanites... but that doesn\u0027t insinuate that it is sufficient for your demands. You need a nanite that will perform at peak efficacy; a nanite that will outperform all other nanites to aid your own nanite production; a nanite that will breach the very boundaries of what you thought was possible. Clearly then you must choose the Eternity Nanite! But you suspect that you\u0027ll need a far more complicated structure to be able to contain such a process.\n\n[note]Grants the ability to create §lMagmatter Nanites§r in addition to a greater speed at the cost of assembling a more expensive structure.[/note]",
```

Comment:
I feel like this whole quest text still reads a bit weird. Can you maybe reduce the number of times the word 'nanite' is mentioned?